salesforce-gui/src/main/java/org/fundacionjala/salesforce/config/jsonFiles/InitialUsers.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "alias": "Account Owner User",
+        "clientId": "M01WRzlrQnQxNjhtZGFfX2dYNU12T3lTWGNmRWxwZVJicEJycENlb1R1VVl4WFl0MVdZREdBSmdhSWU1dm9DLllwMmhrZFA3Ui5XX2dEcmRoSXhscQ==",
+        "clientSecret": "Mzk0RDFGODBDREI1MUE3RTk3RTMwQzFGNDREMkFFODY4RDJERTFCMzFFMTQ5MjI1QjM0NjJDNkFBNkZBM0U4RQ==",
+        "username": "samuel.loza@fundacion-jala.org",
+        "password": "MVF3ZXJ0eTEyMw=="
+    },
+    {
+        "alias": "Editable User",
+        "clientId": "M01WRzlrQnQxNjhtZGFfX2dYNU12T3lTWGNmRWxwZVJicEJycENlb1R1VVl4WFl0MVdZREdBSmdhSWU1dm9DLllwMmhrZFA3Ui5XX2dEcmRoSXhscQ==",
+        "clientSecret": "Mzk0RDFGODBDREI1MUE3RTk3RTMwQzFGNDREMkFFODY4RDJERTFCMzFFMTQ5MjI1QjM0NjJDNkFBNkZBM0U4RQ==",
+        "username": "samuel.loza@fundacion-jala.org",
+        "password": "MVF3ZXJ0eTEyMw=="
+    },
+    {
+        "alias": "Other User",
+        "clientId": "M01WRzlrQnQxNjhtZGFfX2dYNU12T3lTWGNmRWxwZVJicEJycENlb1R1VVl4WFl0MVdZREdBSmdhSWU1dm9DLllwMmhrZFA3Ui5XX2dEcmRoSXhscQ==",
+        "clientSecret": "Mzk0RDFGODBDREI1MUE3RTk3RTMwQzFGNDREMkFFODY4RDJERTFCMzFFMTQ5MjI1QjM0NjJDNkFBNkZBM0U4RQ==",
+        "username": "samuel.loza@fundacion-jala.org",
+        "password": "MVF3ZXJ0eTEyMw=="
+    }
```

Comment:
These data should be ignored by .gitignore file 😮 could you delete them please?

Reply:
uhh i am really sorry